golang/scripts/build.sh

```diff
@@ -10,11 +10,33 @@ root_dirpath="$(dirname "${script_dirpath}")"
 # ==================================================================================================
 #                                             Constants
 # ==================================================================================================
-KURTOSIS_DOCKERHUB_ORG="kurtosistech"
-IMAGE_NAME="kurtosis-lambda-starter-pack"
+IMAGE_NAME="kurtosistech/kurtosis-lambda-starter-pack"
+KURTOSIS_LAMBDA_FOLDER="kurtosis-lambda"
 
 
-# ==================================================================================================
-#                                             Main Logic
-# ==================================================================================================
-docker build -t "${KURTOSIS_DOCKERHUB_ORG}/${IMAGE_NAME}" --progress=plain -f "${root_dirpath}/Dockerfile" "${root_dirpath}"
+# =============================================================================
+#                                 Main Code
+# =============================================================================
+# Build Dockerfile
+# Captures the first of tag > branch > commit
+git_ref="$(cd "${root_dirpath}" && git describe --tags --exact-match 2>/dev/null || git symbolic-ref -q --short HEAD || git rev-parse --short HEAD)"
+if [ "${git_ref}" == "" ]; then
```

Comment:
`git_ref` variable isn't used - the Docker image is just tagged with `IMAGE_NAME` (which means that Docker will use the implicit tag `latest`).

IMO it's actually fine to not use the Git tag (we can let them do this if they want to) so I'd just remove this `git_ref` variable completely

Reply:
removed